app/helpers/request_link_helper.rb

```diff
@@ -57,7 +57,7 @@ def link_data_attributes(callnumber)
   end
 
   def hoover_request_url(document, callnumber)
-    HooverOpenUrlRequest.new(callnumber.library, document, self).to_url
```

Comment:
This helper code is all dead now, but keeping it around for a bit for reference while we transition.  We'll eventually want to 🔪 the entire helper.